### PR TITLE
Chinese (Simplified and Traditional) Translation of Solana Cookbook

### PR DESCRIPTION
**Pull request by lillianrf**
Completed the Chinese (Simplified) translation of the Solana Cookbook (added the Guides and References sections, which were previously missing).

### CODE REVIEW

**Comment by lillianrf** [comment]
Added the traditional translation as well.

File: docs/.vuepress/config.ts [hunk 1]
```diff
@@ -174,6 +174,58 @@ export default defineUserConfig<DefaultThemeOptions>({
...
+              "/zh_t/core-concepts/accounts.md",
+              "/zh_t/core-concepts/programs.md",
+              "/zh_t/core-concepts/transactions.md",
+              "/zh_t/core-concepts/pdas.md",
+            ],
+          },
+          {
+            text: "指南",
+            children: [
+              // '/zh_t/guides/get-program-accounts.md',
```
Comment (jacobcreech) [inline docs/.vuepress/config.ts:203]: Don't forget to uncomment once you've added each. Once this is fixed I'll merge

Comment (lillianrf) [reply docs/.vuepress/config.ts]: Just fixed, thanks.

**Review by jacobcreech — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**